wd_helper.au3

```diff
@@ -1284,6 +1285,15 @@ Func _WD_UpdateDriver($sBrowser, $sInstallDir = Default, $bFlag64 = Default, $bF
 					$sDriverEXE = "msedgedriver.exe"
 			EndSwitch
 
+			; Determine current local webdriver Architecture
+			If FileExists($sInstallDir & $sDriverEXE) Then
+				_WinAPI_GetBinaryType($sInstallDir & $sDriverEXE)
+				Local $bDriverIs64Bit = (@extended = $SCS_64BIT_BINARY)
+				If $sBrowser = 'firefox' Or $sBrowser = 'msedge' Then
+					If $bDriverIs64Bit <> $bFlag64 Then FileDelete($sInstallDir & $sDriverEXE)
+				EndIf
```

Comment:
I think that we should consider either
- dropping the browser check here, or
- changing it to just check <> "chrome"

Reply:
> * dropping the browser check here, or

?

Reply:
> ?

The initial If...Endif could be viewed as unneeded as the function will still work without the "special handling" for Chrome not having a 64 bit version of the webdriver.

Reply:
> * changing it to just check <> "chrome"

done